c7n/resources/mq.py

```diff
@@ -82,6 +83,57 @@ def get_dimensions(self, resource):
         return [{'Name': self.model.dimension,
                  'Value': "{}-1".format(resource['BrokerName'])}]
 
+@MessageBroker.filter_registry.register('vpc')
+class VpcFilter(VpcFilter):
+
+    sg_vpc_dict = {}
+
+    def get_related_ids(self, resources):
+        client = local_session(self.manager.session_factory).client('ec2')
+        related_ids = []
+        for r in resources:
+            sg_ids = r.get('SecurityGroups', [])
+            vpc_id = self.sg_vpc_dict.get(sg_ids[0])
+            if(vpc_id):
+                related_ids.append(vpc_id)
+                continue
+            try:
+                response = client.describe_security_groups(
+                    GroupIds=[sg_ids[0]],
+                )
+                vpc_id = response.get('SecurityGroups')[0].get('VpcId')
+                related_ids.append(vpc_id)
+                self.sg_vpc_dict[sg_ids[0]] = vpc_id
+            except ClientError: 
+                continue 
+        return set(related_ids)
+
+    RelatedIdsExpression = "VpcId"
+
+
+@MessageBroker.filter_registry.register('default-vpc')
+class DefaultVpc(DefaultVpcBase):
+    """ Matches if an mq broker is in the default vpc
+    """
```

Comment:
If your docstring is a one-liner, you can move the closing quotes to be on the same line as the rest of the docstring. If you want to expand on this and add a policy example, you can keep the closing quotes on their own line.